src/stack.js

```diff
@@ -7,32 +7,31 @@ export default class Stack {
   // to push data
   // stack.push(a)
   // stack.push(b)
-  constructor(size, arr){
-    this.size = 0
```

Comment:
great deletion - i was going to say if you're making it into an array, you can use the global length method on js arrays to access the size